### ISSUE TITLE
Feature request: Proper online version

### ISSUE THREAD

**Issue by Heath123** [felixrieseberg/windows95#150]
An online version would be more useful. I know it's a bit of a joke (apparently) but I would probably find that useful. I know about v86 but this is better for some things.

**Comment by Dante-Morrison** [comment]
agreed, it would be useful to have an online version that we can modify with different apps

**Comment by felixrieseberg** [comment]
The online version of this experience already exists and predates this app: https://copy.sh/v86/ (same emulator). This project is deliberately the desktop-app take — local saved state, shared folders, native menus. Closing.
